packages/flutter_app/lib/router/app_routes.dart

```diff
@@ -1,50 +1,72 @@
 import 'package:flutter/material.dart';
 import 'package:go_router/go_router.dart';
 
-import '../features/explore/view/explore_page.dart';
-import '../features/post/view/post_page.dart';
-import '../pages/account/account_page.dart';
-import '../pages/app_info_page/app_info_page.dart';
-import '../pages/dashboard/dashboard_page.dart';
-import '../pages/home_page/home_page.dart';
+import '../pages/initialization_page.dart';
 import '../pages/main/main_page.dart';
-import '../pages/riverpod_example_page/async_notifier_provider_page/async_notifier_provider_page.dart';
-import '../pages/riverpod_example_page/listen_provider_page/listen_provider_page.dart';
-import '../pages/riverpod_example_page/notifier_provider_page/notifier_provider_page.dart';
-import '../pages/riverpod_example_page/provider_page/provider_page.dart';
-import '../pages/riverpod_example_page/riverpod_example_page.dart';
-import '../pages/riverpod_example_page/state_notifier_provider_page/state_notifier_provider_page.dart';
-import '../pages/riverpod_example_page/state_provider_page/state_provider_page.dart';
-import '../pages/riverpod_example_page/stream_provider_page.dart/stream_provider_page.dart';
-import '../pages/settings_page/settings_page.dart';
-import '../pages/theme_selection_page/theme_selection_page.dart';
+import 'branches/branches.dart';
 
 part 'app_routes.g.dart';
 
+@TypedGoRoute<InitializationRoute>(
+  name: InitializationRoute.name,
+  path: InitializationRoute.path,
+)
+class InitializationRoute extends GoRouteData {
+  const InitializationRoute();
+
+  static const String name = '/init';
+  static const String path = '/init';
+
+  @override
+  Page<void> buildPage(BuildContext context, GoRouterState state) {
+    return const NoTransitionPage(child: InitializationPage());
+  }
+}
+
 @TypedStatefulShellRoute<MainShellRouteData>(
   branches: [
     TypedStatefulShellBranch<HomeShellBranchData>(
       routes: [
         TypedGoRoute<HomeRouteData>(
-          path: HomeShellBranchData.path,
+          name: HomeRouteData.name,
+          path: HomeRouteData.path,
```

Comment:
Added `name` to each TypedGoRoute, to be used as the route name sent by NavigatorObserver.